spec/namespaces/ml.yaml

```diff
@@ -300,6 +532,32 @@ components:
             required:
               - status
               - task_id
+    ml.create_model_meta@200:
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              status:
+                type: string
+              model_id:
+                type: string
+            required:
+              - model_id
+              - status
```

Comment:
what status are we referring to here?

Reply:
It is the status of creating the metadata. This is the [reference](https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_commons/model_uploader.py#L124).